docs/yellowpaper/sections/01-introduction/index.tex

```diff
@@ -40,37 +40,30 @@ \subsection{Security Goals}
+These properties assume senders and receivers are honest, but they should hold for any honest senders and receivers of the adversary's choice.
 
+To provide these features, the HOPR protocol builds on top of the Sphinx packet format \cite{sphinxpaper}, the specifics of which are explained in Section \ref{sec:sphinx}.
 
 \subsubsection{Threat Model}
 
-Although we assume that nodes in the HOPR network can communicate reliably, the network must still be protected from malicious actors and node failures. We assume a threat model with byzantine nodes with the ability to either observe all network traffic and launch network attacks or to inject, drop, or delay messages as follows:
-
-\paragraph{A global passive adversary (GPA)} is able to observe all network traffic between users and providers and between mix servers. This adversary is able to observe the entire network infrastructure, launch network attacks such as BGP rerouting  or conduct indirect observations such as load monitoring and off-path attacks. Thus, the GPA is an abstraction that represents many different classes of adversaries able to observe some or all information between network nodes.
-\paragraph{Active adversary:}The adversary has the ability to observe all of the internal state of some corrupted or malicious mix relays. The adversary may inject, drop, or delay messages. She also has access to, and operates, using the secrets of those compromised parties. Furthermore, such corrupted nodes may deviate from the protocol, or inject malformed messages. A variation of this ability is where the mix relay is also the provider node meaning that the adversary additionally knows the mapping between clients and their mailboxes. We say that the provider is corrupt, but is restricted to being honest but curious.
+Although we assume that nodes in the HOPR network can communicate reliably, the network must still be protected from malicious actors and node failures. We assume a threat model with Byzantine nodes with the ability to either observe all network traffic and launch network attacks or to inject, drop, or delay messages as follows:
 
+\paragraph{Global passive adversaries (GPAs)} A global passive adversary (GPA) is a theoretical attacker who can observe the entirety of network traffic passing between users. A GPA is considered passive because their attacks are based on observation alone. A GPA is arbitrarily powerful, and the full abilities of a GPA are unlikely to be manifest in any real-world attacker. Nonetheless, building a network which is GPA-resistant introduced extra security through redundancy and future-proofing.
+\paragraph{Active adversary} In contrast to a GPA, an active adversary controls one or more corrupted nodes within the network and can thus observe the contents of packets passing through these nodes. As an active adversary, these corrupted nodes can be used to inject new packets, delay implementation of the mixing and transmission protocol, or drop packets entirely.
```

Comment:
An active attacker can be modeled as a Dolev-Yao attacker. Such an attacker is able to drop, modify and inject messages between two parties.

In addition, there are corruption attacks (attacker is able to control one node) and collaboration attacks (the attacker is able to control more than one node and/or multiple nodes are collaborating with the attacker).

The HOPR protocol attempts to be secure against passive adversaries but it hasn't been hardened yet against active adversaries. We furthermore assume that collaboration attacks are unlikely due to economical reasons - but this hasn't been proven yet.